### PR TITLE
Task80Allure reporting initial commit

### PR DESCRIPTION
**Pull request by IlyaBaek**

### CODE REVIEW

File: Task10_Auth/src/main/java/Mapper.java [hunk 1]
```diff
@@ -20,4 +23,29 @@
...
+        ObjectMapper objectMapper = new ObjectMapper();
+        String usersString = null;
+        try {
+            usersString = objectMapper.writeValueAsString(users);
+
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+        return usersString;
+    }
```
Comment (EugenBorisik) [inline Task10_Auth/src/main/java/Mapper.java]: I'm surprised, because it is really bad code. Do you really don't know how to make universal method?

**Review by EugenBorisik — CHANGES_REQUESTED** [review]

File: Task10_Auth/src/main/java/Mapper.java (hunk 1 above)
Comment (IlyaBaek) [reply Task10_Auth/src/main/java/Mapper.java]: When I found the answer I've remembered that I've already done it during this course. 

I hope I will never forget that now)

Resolved.

File: Task10_Auth/src/main/java/Mapper.java [hunk 2]
```diff
@@ -20,4 +20,16 @@
             throw new RuntimeException("Can't map to the object of " + clazz.getName());
         }
     }
+
+    public static String UserDtoToString(Object users) {
```
Comment (EugenBorisik) [inline Task10_Auth/src/main/java/Mapper.java]: Method name starts with small letter

**Review by EugenBorisik — APPROVED** [review]

File: Task10_Auth/src/main/java/Mapper.java (hunk 2 above)
Comment (IlyaBaek) [reply Task10_Auth/src/main/java/Mapper.java]: fixed

**Pull request merged**